materials/models/milaco/minecraft_pm/hat_layer.vmt

```diff
@@ -1,5 +1,7 @@
 "vertexlitgeneric"
 {
     "$basetexture" "models\milaco\minecraft_pm\hat_layer"
-    "$smooth" "0"

```

Comment:
are these changes still up to date?

Reply:
Yes, I made the changes after the new minecraft model PR was accepted